netci.groovy

```diff
@@ -3,26 +3,32 @@ import jobs.generation.Utilities
 def project = GithubProject
 def branch = GithubBranchName
 def isPR = true
-def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:WindowsServerCore']
+def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:WindowsServerCore', 'Windows Nano 2016:NanoServer']
 
 platformList.each { platform ->
-	def(hostOS, containerOS) = platform.tokenize(':')
-	def newJobName = Utilities.getFullJobName(project, containerOS, isPR)
+    def(hostOS, containerOS) = platform.tokenize(':')
+    def newJobName = Utilities.getFullJobName(project, containerOS, isPR)
+    def machineLabel = 'latest-or-auto-docker'
 
-	def newJob = job(newJobName) {
-		steps {
-			if (hostOS == 'Windows_2016')
-			{
-				batchFile("powershell -NoProfile -Command .\\build-and-test.ps1")
-			}
-			else
-			{
-				shell("./build-and-test.sh")
-			}
-		}
-	}
+    def newJob = job(newJobName) {
+        steps {
+            if (hostOS == 'Windows_2016')
+            {
+                batchFile("powershell -NoProfile -Command .\\build-and-test.ps1")
```

Comment:
Would it preferable to use the `OS` parameter here also, rather than relying on its default value?